DuplicateFinderEngine/ScanEngine.cs

```diff
@@ -326,17 +339,17 @@ public struct OwnScanProgress {
 				videoFile.FrameSizeInt = bitmapImage.Width + bitmapImage.Height;
 
 				double resizeFactor = 1;
-				if (bitmapImage.Width > 100 || bitmapImage.Height > 100) {
-					double widthFactor = Convert.ToDouble(bitmapImage.Width) / 100;
-					double heightFactor = Convert.ToDouble(bitmapImage.Height) / 100;
+				if (bitmapImage.Width > 300 || bitmapImage.Height > 169) {
+					double widthFactor = Convert.ToDouble(bitmapImage.Width) / 300;
+					double heightFactor = Convert.ToDouble(bitmapImage.Height) / 169;
 					resizeFactor = Math.Max(widthFactor, heightFactor);
 
 				}
 				int width = Convert.ToInt32(bitmapImage.Width / resizeFactor);
 				int height = Convert.ToInt32(bitmapImage.Height / resizeFactor);
 				var newImage = new Bitmap(width, height);
 				using (var g = Graphics.FromImage(newImage)) {
-					g.InterpolationMode = System.Drawing.Drawing2D.InterpolationMode.HighQualityBicubic;
```

Comment:
Did you measured the performance impact? Does it have one? How much? And does it make the image really much better so it's worth it?

Reply:
Since it's only used for display a preview and not actually resizing an image with the expectation of "best looking" I would rather just leave it as it is unless there is a proof if a real benefit of this change